.github/workflows/release.yml

```diff
@@ -0,0 +1,32 @@
+name: release
+
+on:
+  create:
+    tags:
+
```

Comment:
Also, you may have to provide a tag name pattern for it to trigger (i.e. `v*` or `v*.*.**`)

Reply:
makes sense, i will update, thanks for the review